src/main/java/io/cdap/plugin/zendesk/source/batch/http/PagedIterator.java

```diff
@@ -170,10 +170,11 @@ Map<String, Object> getResponseAsMap() throws IOException, InterruptedException
 
   @VisibleForTesting
   String getNextPage(Map<String, Object> responseMap) {
-    // Using cursor based pagination for Tickets and Users
-    if (objectType.equals(ObjectType.TICKETS) || objectType.equals(ObjectType.USERS)) {
-      return (String) responseMap.get(NEXT_CURSOR_URL);
-    }
+    // Commenting cursor based pagination code as Zendesk APIs are having data loss issue with cursor pagination.
+    // Will uncomment this code once issue will get fixed from zendesk side.
```

Comment:
Can you please open a Jira ticket for tracking this so that we remember to uncomment this code when the Zendesk issue is fixed. Also please link to the Jira ticket here in a comment

Reply:
https://cdap.atlassian.net/browse/PLUGIN-1372
Created a ticket and will add the same in code also.